examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -94,18 +34,15 @@ spec:
         service_net_map:
           nova_api_network: internal_api
           nova_libvirt_network: internal_api
-        # CHANGEME
         timesync_ntp_servers:
           - hostname: clock.redhat.com
         # CPU pinning settings
-        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47" # CHANGEME
+        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47"
```

Comment:
note that the compute dfg does not support or recommend the use of isolcpus on non realtime systems.

it is also invalid to include any CPUs that are part of the cpu_share_set in isolcpus regardless of if its a real-time system or not

later we are using cpu_shared_set = 0-3,24-27
so in this case they are not overlaping however

since this is not a realtim host i think isolcpus should be removed for this va and only included in an explicit realtime va instead.

Reply:
Understood.  We will remove the `isolcpus`.

Reply:
@SeanMooney What has changed since 17.1?  Isolcpus has been in use for a long time now for non-realtime NFV deployments.

Reply:
nothing we explcity ask that isolcpus not be used on non realtime host for 17.1 as well.
officially the comput dfg do not recommend the use of isolcpu on non-realtime hosts.
we have had several customer issues related too it form telcos misconfiguing it on non nfv systems and nfv systems